tutorials/introductory/usage.py

```diff
@@ -340,9 +356,9 @@ def my_plotter(ax, data1, data2, param_dict):
 #    use a different backend.  Therefore, you should avoid explicitly calling
 #    `~matplotlib.use` unless absolutely necessary.
 #
-# .. note::
-#    Backend name specifications are not case-sensitive; e.g., 'GTK3Agg'
-#    and 'gtk3agg' are equivalent.
+#
+# The builtin backends
+# --------------------
 #
 # With a typical installation of matplotlib, such as from a
```

Comment:
this paragraph isn't really true anymore now that we have autodetection

Reply:
I don't feel competent enough describing what is currently happening with autodetection. Please push a suggestion or open a ticket to mark this as outdated.